tests/sentry/backup/test_correctness.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import annotations
+
+import io
+import os
+from pathlib import Path
+from typing import NewType, Protocol
+
+from dateutil import parser
+from jsondiff import diff
+from pydantic import FilePath, PositiveInt
+
+from sentry.db.postgres.roles import in_test_psql_role_override
+from sentry.runner.commands.backup import exporter, importer
+from sentry.utils import json
+from sentry.utils.json import JSONData
+from sentry.utils.pytest.fixtures import django_db_all
+
+ComparatorName = NewType("ComparatorName", str)
+ModelName = NewType("ModelName", str)
```

Comment:
I find NewType only useful if it's presenting a public api -- it doesn't look like these are used outside of the file (and every call is just converting to the newtype anyway -- defeating the purpose)

Reply:
My feeling is that the NewTypes make the other parts of the code easier to read, and in any case, we'll likely be exposing this API to other files in the future.

Reply:
I would prefer you remove the newtypes, and the pydantic types -- this isn't using pydantic and `FilePath` isn't correct anyway -- it's a `pathlib.Path`

Reply:
Done.